### PR TITLE
fix(gui): Proper common clean_certificate with validation for all certificate-related forms

### PR DESCRIPTION
**Pull request by themylogin**
Ticket: #23197

### CODE REVIEW

File: gui/system/forms.py [hunk 1]
```diff
@@ -153,6 +153,27 @@ def clean_path_locked(mp):
...
+        instance.cert_chain = True
+
+    try:
+        load_certificate(certificate)
+    except crypto.Error:
+        raise forms.ValidationError(_("CA not in PEM format."))
+
+    #
+    # Should we validate the chain??? Probably
+    # For now, just assume the user knows WTF he is doing
```
Comment (araujobsd) [inline gui/system/forms.py]: Please,  remove the WTF ,  it is not appropriate for an Open Source project!  And if user is doing a mistake we should handle that in a proper way.

Thanks!

**Comment by araujobsd** [comment]
@themylogin  What pep8 says is, 'is not' and not saying 'is None'! And @william-gr you could be a bit more professional, and take the LOL for yourself ;) <--- Advice for life and not for code.

**Comment by themylogin** [comment]
@araujobsd "is not" is a binary operator, we have nothing to compare certificate to here.

**Comment by araujobsd** [comment]
Binary operator is XOR, long discussion for something that is more style than anything else, and it was approved already :)

**Comment by william-gr** [comment]
@araujobsd it was a simple comment disagreeing with you and you came full berserk. This wasnt the first time. You dont have to agree with me, I already said that multiple of times. But if you dont know how to deal with that, keep it to yourself, please.

Also, I dont need advices of life from you,  you can keep them to yourself. Thanks anyway.

**Pull request merged**